src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt

```diff
@@ -1806,17 +1806,26 @@ Column Column() #Column :
     List<String> data = new ArrayList<String>();
     ArrayConstructor arrayConstructor = null;
     Token tk = null;
+    Token subColumnFirst = null;
+    Token subColumnLast = null;
 }
 {
     data = RelObjectNameList()
+    ( LOOKAHEAD(5) "[" subColumnFirst=<S_LONG> "," subColumnLast=<S_LONG> "]" )?
     [ LOOKAHEAD(2) <K_COMMENT> tk=<S_CHAR_LITERAL> ]
     // @todo: we better should return a SEQUENCE instead of a COLUMN
     [ "." <K_NEXTVAL> { data.add("nextval"); } ]
 
-    [ LOOKAHEAD(2) arrayConstructor = ArrayConstructor(false) ]
+    [ LOOKAHEAD(5) arrayConstructor = ArrayConstructor(false) ]
```

Comment:
I am not convinced that this is the right logical order or interpretation. How can we still have an `ArrayConstructor` after having this new `Column` format?